[BCHDCC-95] Hide footer links and List Your Service button

micaelacunhadev opened the pull request:
## Description
<!--- Describe your changes in some detail by answering questions such as: -->
<!---   Why is this change needed? -->
<!---   How does it address the issue?-->

Added 'hidden' class to the 'List Your Services' button and link in the header and to links in the footer to hide them from the page in both the desktop and mobile versions.

## Motivation and Context
<!-- ClickUp ticket IDs are autolinked. So, you only need to list the ticket ID(s) related to this PR -->
[BCHDCC-95](https://app.clickup.com/t/9006094761/BCHDCC-##) - Hide footer links and List Your Service button
<!--- Why is this change required? What problem does it solve? -->
<!--- Link to any ancillary pieces that you had to use to close the ticket and
      think would be interesting or valuable for readers of this PR. This could
      include things like Airbrake exceptions, ruby/rails documentation, blog
      posts, etc.-->
<!--- Describe any difficulties that arose during creation and any tickets that
      were created as a result. If you have thoughts on how to make those
      difficulties less difficult in the future but not as part of this PR,
      write down those thoughts. -->
      
We want the List Your Services button and links in the footer hidden for now.

## Screenshots
<!--- If this is a UI change, put a relevant screenshot(s) here with a 
      description of what is in it. -->
 Desktop view - NOT signed in
<img width="1075" alt="Screenshot 2024-11-14 at 3 17 55 PM" src="https://github.com/user-attachments/assets/3dc50652-0320-4990-8f7f-c0175a49b222">
Desktop view - signed in
<img width="966" alt="Screenshot 2024-11-14 at 3 19 38 PM" src="https://github.com/user-attachments/assets/4b7e9ba2-aacc-4c54-8a4e-c720127cebee">

Mobile view - NOT signed in
<img width="710" alt="Screenshot 2024-11-14 at 3 18 22 PM" src="https://github.com/user-attachments/assets/ce523e00-4b7b-4d4e-a4f8-a77b159658f8">

Mobile view - signed in
<img width="710" alt="Screenshot 2024-11-14 at 3 19 14 PM" src="https://github.com/user-attachments/assets/a2612497-f40c-431d-a30c-df9dd657a0ff">

## Migrations to Run: No

## Tests to Run
None


## Internal Checklist
<!--- Go over all the following points and make sure you feel comfortable you covered them. -->
- [ ] My code follows the code style of this project (https://rubystyle.guide/).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

micaelacunhadev committed:
Added 'hidden' class to the 'List Your Services' button and link in the header and to links in the footer

anna-dorigo reviewed (changes requested):
Since we are leaving the items there (not deleting only hiding them) for future use, I think would be good to add a comment for future developers to know why there are there but hidden all the time.

micaelacunhadev committed:
Added comments on the hidden footer links and 'List Your Services' button

micaelacunhadev commented:
> Since we are leaving the items there (not deleting only hiding them) for future use, I think would be good to add a comment for future developers to know why there are there but hidden all the time.

@anna-dorigo good point! I just added comments to where I added the `hidden` class.

anna-dorigo commented on app/views/shared/_footer.html.haml:
```diff
@@ -5,7 +5,9 @@
 
-    .footer-links{ role: 'navigation', aria: { label: 'Footer Navigation' } }
+    -# hiding footer links for the time being (11/15/2024)
+    -#  please remove this commment block when revisiting this
```
```suggestion
    -# please remove this comment block when revisiting this
```

anna-dorigo commented on app/views/shared/_header.html.haml:
```diff
@@ -10,7 +10,9 @@
     %ul.desktop-only
-      %li
+      -# hiding the 'List Your Services' button for the time being (11/15/2024)
+      -#  please remove this commment block when revisiting this
```
```suggestion
      -# please remove this comment block when revisiting this
```

anna-dorigo commented on app/views/shared/_header.html.haml:
```diff
@@ -47,7 +49,9 @@
           %img{ src: asset_path('language_flags/spanish_language_flag.png'), alt: 'Spanish' }
-      %li
+      -# hiding the 'List Your Services' link for the time being (11/15/2024)
+      -#  please remove this commment block when revisiting this
```
```suggestion
      -# please remove this comment block when revisiting this
```

anna-dorigo committed:
Update app/views/shared/_footer.html.haml

anna-dorigo committed:
Update app/views/shared/_header.html.haml

anna-dorigo committed:
Update app/views/shared/_header.html.haml

anna-dorigo reviewed (approved):
This looks good! I just fixed a typo and an extra white space.

micaelacunhadev committed:
BCHDCC-95 Reverted changes unrelated to this branch

micaelacunhadev committed:
Merge branch 'BCHDCC-95-hide-footer-links-and-list-your-service-button' of github.com:bchd/bahc-ohana-api into BCHDCC-95-hide-footer-links-and-list-your-service-button
